token.py

```diff
@@ -0,0 +1,19 @@
+from itsdangerous import URLSafeTimedSerializer
+
+import app
+
+def generate_confirmation_token(email):
+    serializer = URLSafeTimedSerializer(app.config['SECRET_KEY'])
+    return serializer.dumps(email, salt = app.config['SECURITY_PASSWORD_SALT'])
+
+def confirm_token(token, expiration: object = 3600):
+    serializer = URLSafeTimedSerializer(app.config['SECRET_KEY'])
+    try:
+        email = serializer.loads(
+            token,
+            salt = app.config['SECURITY_PASSWORD_SALT'],
+            max_age = expiration
+        )
+    except:
+        return False
+    return email
```

Comment:
We don't use this file either. I forgot to remove it, so we should in the near future.